src/components/CopyToClipboard/index.ts

```diff
@@ -1 +1,2 @@
 export * from './CopyToClipboard';
+export * from './types';
```

Comment:
This will export some private types from package. `CopyToClipboardStatus ` is the only public here imo.